Project.toml

```diff
@@ -0,0 +1,17 @@
+name = "LibPNG"
+uuid = "108bb7d8-ee5e-11e9-2f8f-6ba5bcecef1c"
+authors = ["msb <mandeep.bahal@roames.com.au>"]
```

Comment:
Would be sensible to list Josh Christie in the authors list I think!

Reply:
I dont mind either way! :)

Reply:
Agreed! I didn't know I was added in the list either.